movie-showcase/src/routes/+page.svelte

```diff
@@ -0,0 +1,25 @@
+<div class="select">
```

Comment:
The name select is a bit confusing, because that would imply that this is some kind of `<select>` replacement. Maybe 'menu' is a better name.